includes/Commerce/Orders.php

```diff
@@ -96,7 +97,55 @@ public function update_local_order( Order $remote_order, \WC_Order $local_order
 	 */
 	public function update_local_orders() {
 
-		// TODO: implement
+		$page_id = facebook_for_woocommerce()->get_integration()->get_facebook_page_id();
+
+		try {
+
+			$response = facebook_for_woocommerce()->get_api()->get_new_orders( $page_id );
+
+		} catch ( SV_WC_API_Exception $exception ) {
+
+			facebook_for_woocommerce()->log( 'Error fetching Commerce orders from the Orders API: ' . $exception->getMessage() );
+
+			return;
+		}
+
+		$remote_orders = $response->get_orders();
+
+		foreach ( $remote_orders as $remote_order ) {
+
+			$local_order = $this->find_local_order( $remote_order->get_id() );
+
+			try {
+
+				if ( empty( $local_order ) ) {
+					$local_order = $this->create_local_order( $remote_order );
+				} else {
+					$local_order = $this->update_local_order( $remote_order, $local_order );
+				}
+
+			} catch ( \Exception $exception ) {
+
+				if ( ! empty( $local_order ) ) {
+					// add note to order
+					$local_order->add_order_note( 'Error updating local order from Commerce order from the Orders API: ' . $exception->getMessage() );
+				} else {
+					facebook_for_woocommerce()->log( 'Error creating local order from Commerce order from the Orders API: ' . $exception->getMessage() );
+				}
+
+				return;
+			}
+
+			if ( ! empty( $local_order ) && Order::STATUS_CREATED === $remote_order->get_status() ) {
+
+				// acknowledge the order
+				try {
+					facebook_for_woocommerce()->get_api()->acknowledge_order( $remote_order->get_id(), $local_order->get_id() );
```

Comment:
this will also need the page access token

Reply:
Fixed in https://github.com/facebookincubator/facebook-for-woocommerce/pull/1520/commits/5c72c5e8b4d3b62a9761eddd85f85824051f82e9